run-integration-tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Please remove this. Integration test should not required complex setup

Reply:
@b00f Mo I think this bash script is a good idea to run tests instead of letting developers run each command by themselves.
If a person has a complete understanding of the project, of course he has the choice of his own testing method, but for someone who has zero-knowledge on the project (not yet!), this will be a good start.

Reply:
> someone who has zero-knowledge on the project (not yet!), this will be a good start.

For those we can update README and keep project tree clean for us...